node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Could you remove this file, the node_modules folder, and the yarn.lock?

Reply:
Removed both yarn files. Should I also add them to the .gitignore?